libs/data-access/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './lib/api/concept';
 export * from './lib/context/concept';
```

Comment:
Kanskje vi bør samle alt som har med concept å gjøre under lib. Så isteden for lib/api/concept og lib/context/concept blir det lib/concept/api og lib/concept/context.

Reply:
God ide, vil det ikke kræsje med nx sin default måte oprette bibliotek komponenter på?

Reply:
Ikke vis project er concept og component er api/context.

Reply:
Jeg har gjort det. Bare gjør oppmerksom på at nå må vi lage en api-mappe under hver app i stedet for å ha kun èn api med appene under. Det blir da sånn:
- concpt  > api, 
- dataset > api, 
osv.